src/app/inicio/page.tsx

```diff
@@ -1,29 +1,71 @@
 'use client'
+import React, {useState} from 'react';
+import { QueryClient, QueryClientProvider, useMutation, useQuery } from "react-query";
+import OpenAI from 'openai';
+import { userInputStore } from '@/store/inputStore';
 
-import { QueryClient, QueryClientProvider, useQuery } from "react-query";
+const openai = new OpenAI({
+  apiKey: process.env.NEXT_PUBLIC_OPENAI_API_KEY, 
+  dangerouslyAllowBrowser: true
+});
 
 export default function Home() {
-  const {data, status} = useQuery('data', dataFetch);
+  //Zudstand for the input of the user
+  const { userInputs} = userInputStore();
```

Comment:
you can unify this by extracting both members from the same hook.
`const [userInputs, updateUserInput] = userInputStore((state) => [state.userInputs, state.updateUserInput]);`

Reply:
Great, thank you. Just implemeted it